network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -55,17 +55,16 @@
     <div class="tw-row">
       <div class="tw-w-full">
         <div id="sticky-bar" class="creepiness-slider bg-white text-center tw-justify-center creep-o-meter-moved search-active">
-          <div class="creep-o-meter-information">
-            <p class="speech-bubble-container">
-              <span class="speech-bubble tw-bg-gradient-to-t tw-from-purple-05 tw-to-blue-05">
-                <img class="hook" src="{% static "_images/buyers-guide/icon-bubble-hook.svg" %}">
-                <span class="text">
+          <div class="creep-o-meter-information tw-flex tw-items-center tw-whitespace-nowrap">
+            <div class="speech-bubble-container tw-inline-block tw-mr-4">
+              <div class="speech-bubble tw-relative tw-inline-block tw-whitespace-normal tw-text-left tw-bg-gradient-to-t tw-from-purple-05 tw-to-blue-05">
```

Comment:
Do we need the `tw-text-left` class? Is it needed for an override?

Reply:
Yup! A parent of this has a text align center styling.